pallets/creator-staking/src/functions.rs

```diff
@@ -434,30 +434,85 @@ impl<T: Config> Pallet<T> {
         withdrawable_amounts_by_creator
     }
 
-    pub fn available_claims_by_backer(
-        backer: T::AccountId,
-    ) -> Vec<(CreatorId, u32)> {
-        let mut available_claims_by_creator = BTreeMap::new();
+    pub fn available_claims_by_backer(backer: T::AccountId) -> Vec<(CreatorId, u32)> {
+        let mut available_claims_by_creator = Vec::new();
```

Comment:
<img width="417" alt="image" src="https://github.com/dappforce/subsocial-parachain/assets/39906111/18c7058b-1cb6-44c4-bc9e-4e43b1178a5f">
I get confused when I see by_baker and by_creator together.

So maybe better: `available_backer_claims_by_creator`